owlbot.py

```diff
@@ -66,35 +77,15 @@
 # Generate _pb2.py files and format them
 s.shell.run(["nox", "-s", "generate_protos"])
 
-# Clean up
+# Clean up the folders for dependencies which are shipped via `googleapis-common-protos`
```

Comment:
You have to keep this in sync with the lines above, right? So maybe capturing the protos in a list and running through the list in both places

```python
# Above:
common_apis = ["google/api","google/iam/v1","google/rpc/context","google/type"]
_ = [os.makedirs(dir, exist_ok=True) for dir in common_apis]

# Here:
_ = [shutil.rmtree(dir) for dir in common_apis]
```

Reply:
Fixed in https://github.com/googleapis/python-access-context-manager/pull/201/commits/eda2523b25c082bb653f23c33ee4b8820ea47228